addons/google_calendar/i18n/google_calendar.pot

```diff
@@ -400,6 +400,21 @@ msgid ""
 "event permissions set on Google Calendar."
 msgstr ""
 
+#. module: google_calendar
+#. odoo-python
+#: code:addons/google_calendar/models/calendar.py:0
+msgid ""
+"Success! The selected events are now synchronized with Google Calendar."
+msgstr ""
+
+#. module: google_calendar
+#. odoo-python
+#: code:addons/google_calendar/models/calendar.py:0
+msgid ""
+"Please try again later, an error occurred trying to synchronize the "
+"selected events."
+msgstr ""
+
```

Comment:
You don't need to update translations in master. They will be regenerated after the freeze.